anndata/tests/helpers.py

```diff
@@ -256,19 +256,17 @@ def gen_adata(
         awkward_ragged=gen_awkward((12, None, None)),
         # U_recarray=gen_vstr_recarray(N, 5, "U4")
     )
-    with warnings.catch_warnings():
-        warnings.simplefilter("ignore", ExperimentalFeatureWarning)
-        adata = AnnData(
-            X=X,
-            obs=obs,
-            var=var,
-            obsm=obsm,
-            varm=varm,
-            layers=layers,
-            obsp=obsp,
-            varp=varp,
-            uns=uns,
-        )
+    adata = AnnData(
+        X=X,
+        obs=obs,
+        var=var,
+        obsm=obsm,
+        varm=varm,
+        layers=layers,
+        obsp=obsp,
+        varp=varp,
+        uns=uns,
+    )
```

Comment:
```suggestion
        with warnings.catch_warnings():
        warnings.simplefilter("ignore", ExperimentalFeatureWarning)
        adata = AnnData(
            X=X,
            obs=obs,
            var=var,
            obsm=obsm,
            varm=varm,
            layers=layers,
            obsp=obsp,
            varp=varp,
            uns=uns,
        )
```

Would you be against just not touching this part?

Reply:
I don’t see why we should double-ignore it

Its existence implies that this filter exists for a reason. But not important.